README.md

```diff
@@ -46,6 +46,17 @@ you shouldn't as you know)
 
 The build result will be a Pentaho package located in ```target```.
 
+#### Packaging / Distributing it
+
+Packages can be build by using the following command:
```

Comment:
please change to "built" instead of "build"

Reply:
@ddiroma Thanks :-) I have fixed that now